Makefile

```diff
@@ -21,3 +21,9 @@ db:
 	createuser --superuser pointfreeco || true
 	createdb --owner pointfreeco pointfreeco_development || true
 	createdb --owner pointfreeco pointfreeco_test || true
+
+install-cmark:
+	apt-get -y install cmake
+	git clone https://github.com/commonmark/cmark
+	make -C cmark INSTALL_PREFIX=/usr
```

Comment:
Does this require `sudo`? Should we be using `/usr/local`?

Reply:
`sudo` is required to install it in `/usr`. the default is `/usr/local` but then when compiling on travis it couldn't find `cmark.h`. putting it in `/usr` seemed to fix. 

I do the same in the other PR https://github.com/pointfreeco/pointfreeco/pull/100/files#diff-354f30a63fb0907d4ad57269548329e3R15

here's an example of what travis does when installing into `/usr/local`:

https://travis-ci.org/pointfreeco/pointfreeco/builds/326596574?utm_source=github_status&utm_medium=notification

```
Linking ./.build/x86_64-unknown-linux/debug/PointFreePackageTests.xctest
/home/travis/build/pointfreeco/pointfreeco/.build/x86_64-unknown-linux/debug/PointFreePackageTests.xctest: error while loading shared libraries: libcmark.so.0.28.3: cannot open shared object file: No such file or directory
```

any ideas?